changelog.html

```diff
@@ -43,6 +43,12 @@
 <h1>
 Monitoring Plugin Changelog
 </h1>
+    
+<p><b>1.8.2</b> -- November 25, 2019</p>
+<ul>
+    <li>[<a href='https://github.com/igniterealtime/openfire-monitoring-plugin/issues/60'>Issue #53</a>] - Java11 ClassNotFoundException when deploying.</li>
+    <li>Fix Syntax in SQL Statement for SQL Server in PaginatedMucMessageQuery</li>
```

Comment:
I hate to nag, but can you please provide a link (`a href=`) to the issue on github?

Reply:
of cource.. wait a moment ... done